gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/validator/CronValidator.java

```diff
@@ -43,7 +44,7 @@ public void validate(ValidatorContext ctx)
     Object value = element.getValue();
     String str = String.valueOf(value);
 
-    if (!CronExpression.isValidExpression(str))
+    if (!StringUtils.isEmpty(str) && !CronExpression.isValidExpression(str))
```

Comment:
Why this change? The original intention is that if a schedule is set then it should have a valid cron expression. Run once occurs when a schedule is not set.

Reply:
cronSchedule is compulsory field in Schedule, so it cannot be skipped, it can only be written as empty. Do you want to make it optional in Schedule.pdsc?

Reply:
We should not allow an empty cron expression. If the user wants to configure a run-once job then they should not set a schedule.